### PR TITLE
Cleanup styles

### PR DESCRIPTION
**Pull request by breezyboa**
Addresses #83

### CODE REVIEW

**Comment by breezyboa** [comment]
@rishimohan _a Can you review this?

File: app/assets/stylesheets/client/app.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
+@import "vars";
+
+#acehelp-hook{
```
Comment (rishimohan) [inline app/assets/stylesheets/client/app.scss]: Add a space after the name, `#acehelp-hook {`, same for all the selectors below.

File: app/assets/stylesheets/client/app.scss [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        color: #fff;
+        font-size: 80px;
+        box-shadow: 0 0 30px 0px rgb(153, 153, 153);
+        cursor: pointer;
+        animation: mini-button-pop-in 400ms 1;
+        animation-timing-function: cubic-bezier(0.68, 0.55, 0.265, 1.55);
+        transition: transform .2s ease-in-out, box-shadow .5s ease-in-out;
+
+        &:hover{
+            transform: scale(1.1);
```
Comment (rishimohan) [inline app/assets/stylesheets/client/app.scss]: For cross-browser compatibility, add : 
```
-ms-transform: scale(1.1);
-webkit-transform: scale(1.1);
transform: scale(1.1);
```

Do the same for all the transform properties.

File: app/assets/stylesheets/client/app.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    .close-button{
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 35px;
+        line-height: 0;
+        cursor: pointer;
+
+        svg {
+            width: 20px; height: 20px; fill: #fff; position: absolute; top: 50%; transform: translateY(-50%);
```
Comment (rishimohan) [inline app/assets/stylesheets/client/app.scss]: Make the property in individual lines just like for all the above selectors.

```
width: 20px; 
height: 20px;
...
...
```

**Comment by rishimohan** [comment]
@breezyboa _a Please update with mentioned changes.

**Comment by breezyboa** [comment]
@rishimohan _a changes done 
404s happen on localhost :( 
also, apis give a CORS error on PR builds. Both these issues need to be investigated separately

**Comment by rishimohan** [comment]
@breezyboa _a Looks good to me.

**Pull request merged**